src/System.Runtime.InteropServices.WindowsRuntime/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcore50aot-Windows_NT;
+      netstandard1.3-Windows_NT;
```

Comment:
Don't you need to add a OS for the other netstandard1.3 build configuration? If you don't I suspect it might build on windows twice when we start doing our build-all traversal.